macros/edr/system/system_utils/upload_information.sql

```diff
@@ -0,0 +1,26 @@
+{% macro upload_information() %}
+  {% set edr_cli_run = elementary.get_config_var('edr_cli_run') %}
+  {% if not execute or edr_cli_run %}
+    {{ return('') }}
+  {% endif %}
+
+  {% set identifier = 'information' %}
+  {% set relation = elementary.get_elementary_relation(identifier) %}
+  {% if not relation %}
+    {{ return('') }}
+  {% endif %}
+
+  {% if results %}
+    {% if elementary.get_result_node(identifier) %}
```

Comment:
Is there a reason why we will enable uploading it not only from an on-run-end hook?

Reply:
[Not anymore](https://github.com/elementary-data/dbt-data-reliability/pull/121/commits/b8f7c745f04f2e051f44ad9436190be52f98574f).